modules/ROOT/pages/policy-visualization.adoc

```diff
@@ -0,0 +1,51 @@
+= Policy Visualization
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+== About the Policy Visualization
+
+[NOTE]
+When deploying a Mule app, you must ensure that you are using a version of Mule that supports the Policy Visualization. Learn more about which xref:setup.adoc#select-a-deployment-type[runtimes are supported].
+
+The Policy Visualization enables you to see the relationship between Mule app deployments and the APIs they are implementing and which policies have been applied to those APIs. Data about which policies have been applied to your APIs comes from API Manager.
+
+There are two modes available within the Policy Visualization: _overview_ and _compliance_. Both modes show you your deployed APIs, but each shows which policies have been applied in a different manner.
```

Comment:
```suggestion
The Policy Visualization feature has two modes: _overview_ and _compliance_. Both modes show your deployed APIs, but each in a different manner:
```
Avoid beginning sentences with "There are," which hides the subject of the sentence.